src/utils/resources/vm/hooks/useProvisioningPercentage.tsx

```diff
@@ -0,0 +1,51 @@
+import { DataVolumeModelGroupVersionKind } from '@kubevirt-ui/kubevirt-api/console';
+import type { V1beta1DataVolume } from '@kubevirt-ui/kubevirt-api/containerized-data-importer/models';
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { getNamespace } from '@kubevirt-utils/resources/shared';
+import { getVolumes } from '@kubevirt-utils/resources/vm';
+import { useK8sWatchResources, WatchK8sResource } from '@openshift-console/dynamic-plugin-sdk';
+import { printableVMStatus } from '@virtualmachines/utils';
+
+export type NameWithPercentages = { [name in string]: string };
```

Comment:
if its export not by mistake and consumed by another component, please add it to types file